main/management/commands/notifications.py

```diff
@@ -0,0 +1,58 @@
+from datetime import timedelta
+
+from django.core.management.base import BaseCommand
+from django.db import models
+from django.utils import timezone
+
+from cdh.mail.utils import send_template_email
+
+from experiments.models import ParticipantSession
+
+
+class Command(BaseCommand):
+    help = 'Sends usage statistics to admins'
+
+    def add_arguments(self, parser):
+        parser.add_argument('email', type=str)
+
+    def handle(self, *args, **options):
+        last_sunday = timezone.now() - timedelta(days=timezone.now().weekday() + 1)
+        range_from = last_sunday - timedelta(days=7)
+        range_to = last_sunday
+
+        new_sessions = ParticipantSession.objects.filter(
+            date_updated__gte=range_from,
+            date_updated__lte=range_to
+        )
+
+        per_day = new_sessions.annotate(
+            day=models.functions.TruncDate('date_updated')
+        ).values(
+            'day'
+        ).annotate(
+            count=models.Count('day')
+        ).values_list('day', 'count').order_by('day')
```

Comment:
This probably works just fine, but should be tested against a mysql database. 

I've seen too many DB differences to just trust Django on this